setup.py

```diff
@@ -204,7 +203,7 @@ def build_extension(self, ext: CMakeExtension):
     "install_requires": requirements,
```

Comment:
Cannot comment on the above line, but I wonder if we need to update line 202 to reflect which backend. Maybe it is also fine to delete it.

Reply:
I will delete it for now.